lib/utils/setCookie.js

```diff
@@ -18,13 +18,17 @@ module.exports = function setCookie (storage, name, value) {
     local: storage.domain === 'localhost',
     custom: storage.customDomain
   }
+
+  var sameSite = 'None'
+
   var expires = new Date()
   expires.setSeconds(expires.getSeconds() + clone.expires)
 
   // When it's localhost, an IP, or custom domain, set the cookie directly
-  if (is.ip || is.local || is.custom) {
-    clone.domain = is.local ? null : clone.domain
-    cookie.setItem(name, value, expires, clone.path, clone.domain)
+  if (is.local) {
+    cookie.setItem(name, value, expires, clone.path, null)
```

Comment:
IMO, for consitency with the later cookie.setItem call:
`cookie.setItem(name, value, expires, clone.path, false, 'None')`

Reply:
Unfortunately, 'Secure' is always set to `true` when setting sameSite to 'None', for the local we can ignore this. IMHO it is for testing only.